visualizations/rhods-notebooks/horreum_lts_store.py

```diff
@@ -161,27 +161,27 @@ def build_lts_payloads() -> dict:
 
         start_time: datetime.datetime = results.start_time
         end_time: datetime.datetime = results.end_time
-
-        output = {
+        payload = {
             "$schema": "urn:rhods-matbench-upload:3.0.0",
             "data": {
                 "users": _decode_users(results),
                 'rhods_version': results.rhods_info.version,
                 'ocp_version': results.sutest_ocp_version,
                 'metrics': _gather_prom_metrics(entry),
                 'thresholds': results.thresholds,
-                'config': results.test_config.yaml_file
+                'config': results.test_config.yaml_file,
+                "cluster_info": _parse_entry(entry.results.rhods_cluster_info),
```

Comment:
I guess `_parse_entry` could have a more meaningful name, explaining that it transforms into dictionaries 

but is this field `cluster_info` actually used in Horreum side? looking at the content, I'm not sure it's relevant to include it in the LTS payload

Reply:
hum, makes sense. I guess it's the control plane nodes that matter most, but still 🤔 
I guess we can keep it then, we'll have to know this list as well in horreum to track the idleness of the control plane nodes